feature-libs/user/profile/components/registration-verification-token-form/verify-register-verification-token-form.component.ts

```diff
@@ -241,10 +241,12 @@ export class RegistrationVerificationTokenFormComponent implements OnInit {
 
   protected onRegisterUserSuccess(): void {
     if (
-      this.authConfigService.getOAuthFlow() ===
-      OAuthFlow.ResourceOwnerPasswordFlow
+      [
+        OAuthFlow.ResourceOwnerPasswordFlow,
+        OAuthFlow.AuthorizationCode,
+      ].includes(this.authConfigService.getOAuthFlow())
     ) {
-      this.router.go('login');
+      this.router.go({ cxRoute: 'login' });
```

Comment:
Yeah but then still should not we threat it as breaking change and handle with feature flag?
In that file we have also another example of usage it `this.router.go(['/login/register']);`

Reply:
to be safe I added the feature flag condition.